jinahub/segmenters/VADSpeechSegmenter/vad_speech_segmenter.py

```diff
@@ -0,0 +1,148 @@
+__copyright__ = 'Copyright (c) 2020-2021 Jina AI Limited. All rights reserved.'
+__license__ = 'Apache-2.0'
+
+from pathlib import Path
+from typing import Optional, Dict, Tuple
+
+import torch
+import torchaudio
+import numpy as np
+import librosa as lr
+from jina import Executor, Document, DocumentArray, requests
+from jina.excepts import BadDocType
+
+
+class VADSpeechSegmenter(Executor):
+    """
+     Segment the speech audio using Silero's Voice Activity Detector (VAD)
+
+    :param normalize: a bool to specify whether to normalize the audio
+     by the sample rate
+    :param dump: a bool to specify whether to dump the segmented audio
+    :param repo: default repo name of silero-vad
+    :param model: default model name of silero-vad
+
+    """
+
+    TARGET_SAMPLE_RATE = 16000
+
+    def __init__(
+        self,
+        normalize: bool = False,
+        dump: bool = False,
+        repo: str = 'snakers4/silero-vad',
+        model: str = 'silero_vad',
```

Comment:
Do we have other options for `repo` and `model`? If not, please remove them from arguments

Reply:
No, we don't have other options at the moment. These arguments are now removed [here](https://github.com/jina-ai/executors/pull/97/commits/752f958d7e44f07d635699131e1a7a43febce5ab).